### ISSUE TITLE
Help me inference model!

### ISSUE THREAD

**Issue by tks1998** [KevinMusgrave/pytorch-metric-learning#338]
Currently, I use arc face loss function + resnet18. 
In the inference phase, I put an image in model resnet18 and get embedding, Do I have to put images into the metric learning to find embedding with the best hyperplane?

**Comment by KevinMusgrave** [comment]
If I understand correctly, you want to use the classifier learned by ArcFaceLoss, rather than do k-nearest-neighbors classification using the embedding space.

First initialize ArcFaceLoss and load the saved weights:
```python
from pytorch_metric_learning.losses import ArcFaceLoss
loss_func = ArcFaceLoss(num_classes, embedding_size)
loss_func.load_state_dict(torch.load(<path_goes_here>))
```

Then use LogitGetter
```python
from pytorch_metric_learning.utils.inference import LogitGetter
logit_getter = LogitGetter(classifier = loss_func, transpose = True)

embeddings = model(imgs)
logits = logit_getter(embeddings)
predictions = torch.argmax(logits, dim=1)
```

I admit it's not the best design. I'm planning to move this functionality into the loss function (see issue https://github.com/KevinMusgrave/pytorch-metric-learning/issues/256)

**Comment by tks1998** [comment]
@KevinMusgrave So many tks :D

**Comment by KevinMusgrave** [comment]
If you want to compare two images, then leave out ArcFaceLoss and LogitGetter. Use the embeddings directly from your resnet model.

```python
e1= model(imgs1)
e2= model(imgs2)
if distance(e1, e2) < threshold:
    ...
```

**Comment by tks1998** [comment]
Thank great framework!
